cheatsheets/Vulnerable_Dependency_Management_Cheat_Sheet.md

```diff
@@ -0,0 +1,233 @@
+
+**Note:** 
+
+In the rest of the cheat sheet, when we refer to *development team* then we assume that the team contain an member with application security skills or can ask help to someone in the company having these kind of skills to analyse the vulnerability impacting the dependency.
+
+# Remark about the detection
+
+It's important to keep in mind the different way in which a security issue is handled after is discovery. 
+
+*Way 1: [Responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure)*
+
+Someone find a issue on a component and collaborate with the provider leading, most of the time, to the creation of a [CVE](https://en.wikipedia.org/wiki/Common_Vulnerabilities_and_Exposures) (sometime a specific vulnerability identifier to the provider is created but generally a CVE identifier is preferred) associated to the issue allowing the public referencing of the issue as well as, sometime, the available fixation/mitigation (sometime provider [do not want to fix the issue](https://www.excellium-services.com/cert-xlm-advisory/cve-2019-7161/)). 
+
+In this way, the exploitation code/method is only provided to the provider of the component and not publicly disclosed.
+
+Here the vulnerability is always referenced in the [CVE global database](https://nvd.nist.gov/vuln/data-feeds) used, generally, by the detection tools as one of the several input source used.
+
+*Way 2: [Full disclosure](https://en.wikipedia.org/wiki/Full_disclosure_(computer_security)#Full_disclosure)*
+
+Someone find a issue on a component and release all the information including exploitation code/method on site like [Full Disclosure mailing list](https://seclists.org/fulldisclosure/), [Exploit-DB](https://www.exploit-db.com)...
+
+Here a CVE is not always created then the vulnerability is not always in the CVE global database causing the detection tools to be potentially blind about this issue unless they use other input sources.
+
+Conclusion, it's important to ensure, during the selection process of a vulnerable dependency detection tool, that this one:
+* Use several reliable input sources in order to handle the both vulnerability disclosure ways.
+* Support for flagging a issue raised on component as *[false-positive](https://www.whitehatsec.com/glossary/content/false-positive)*, see why later in this cheat sheet...
+
+# Remark about the security issue handling decision
+
+When a securit issue is detected, it's also possible to decide to accept the risk represented by the security issue. However, this decision must be taken by the [Chief Risk Officer](https://en.wikipedia.org/wiki/Chief_risk_officer) (fallback possible to [Chief Information Security Officer](https://en.wikipedia.org/wiki/Chief_information_security_officer)) of the company based on technical feedback from the development team that have analysed the issue (see the *[Cases](#-cases)* section) as well as the CVE's [CVSS](https://www.first.org/cvss/user-guide) score indicators.
```

Comment:
spelling (add **y** to securit): securit**y**,
it's also possible - **also** is unnecessary

Reply:
Fixed in coming commit.